components/camera/server.go

```diff
@@ -105,19 +105,19 @@ func (s *serviceServer) GetImages(
 	defer span.End()
 	cam, err := s.coll.Resource(req.Name)
 	if err != nil {
-		return nil, errors.Wrap(err, "camera server GetImages had an error getting the camera component")
+		return nil, errors.Join(err, errors.New("camera server GetImages had an error getting the camera component"))
 	}
 	// request the images, and then check to see what the underlying type is to determine
 	// what to encode as. If it's color, just encode as JPEG.
 	imgs, metadata, err := cam.Images(ctx)
 	if err != nil {
-		return nil, errors.Wrap(err, "camera server GetImages could not call Images on the camera")
+		return nil, errors.Join(err, errors.New("camera server GetImages could not call Images on the camera"))
```

Comment:
@nicksanford a heads up, let me know if the error formats are checked in the module or tests in the module.